vss-extension.json

```diff
@@ -108,12 +108,20 @@
                       "defaultValue": "US",
                       "possibleValues": [
                         {
-                          "value": "us",
-                          "displayValue": "US"
+                            "value": "us",
+                            "displayValue": "US"
```

Comment:
Might be worth indicating where the default region is in the displayValue i.e "US (East 1)", then "US (East 2)", "US (West 2)" is there a US West 1??

Reply:
Ok will add that. There doesn't appear to be a US West 1